### PR TITLE
Added median grade to Grade Distribution view

### PR DESCRIPTION
**Pull request by justin0022**
Addresses #543

### CODE REVIEW

File: assets/src/containers/GradeDistribution.js [hunk 1]
```diff
@@ -34,21 +34,30 @@ function GradeDistribution (props) {
...
+              'My Grade', <strong>{gradeData[0].current_user_grade
+                ? `${roundToOneDecimcal(gradeData[0].current_user_grade)}%`
+                : 'There are no grades yet for you in this course'}</strong>
+            ],
+            [
+              'Average Grade',
+              <strong>{roundToOneDecimcal(average(grades))}%</strong>
+            ],
+            [
+              'Median Grade',
```
Comment (pushyamig) [inline assets/src/containers/GradeDistribution.js]: I did not notice this earlier, BE is sending the average grade already and can also send median. I can round the decimal to one as you are doing it here. This is how it is going to look. I think UI don't need to do the calculation can simply grab the value, what do you say?
```
[  
   {  
      "current_grade":95.9,
      "current_user_grade":null,
      "graph_upper_limit":110,
      "tot_students":35,
      "grade_avg":95.0,
      "grade_median":95.9
   }
]
```

Comment (pushyamig) [reply assets/src/containers/GradeDistribution.js]: This is a simple change at the BE
```
 average_grade = df['current_grade'].mean().round(1)
 median_grade = df['current_grade'].median().round(1)
 df['grade_avg'] = average_grade
 df['grade_median']=median_grade
```

Comment (justin0022) [reply assets/src/containers/GradeDistribution.js]: I'm happy with backend doing calculations, however right now for the histogram we need the actual grades sent as well, so it's easy enough to calculate average and median. When we address #542 then we won't be sending all grades, so the backend calculation will be needed.

Comment (pushyamig) [reply assets/src/containers/GradeDistribution.js]: ok.

**Pull request merged**